### ISSUE TITLE
Remove lightbox from the styleguide

### ISSUE THREAD

**Issue by zufrieden** [swiss/styleguide#212]
Document propose to implement another lightbox. Our recommendation as told since the beginning of the project. Get rid of lightbox would be a better idea.

![screenshot 2014-08-08 16 27 07](https://cloud.githubusercontent.com/assets/451236/3857853/1b86ae0c-1f08-11e4-9a3d-16c3801eb529.png)

![screenshot 2014-08-08 16 26 53](https://cloud.githubusercontent.com/assets/451236/3857850/1124d024-1f08-11e4-88ce-5a6dadfc867f.png)

- [ ] update documentation and HTML page exemples according to new bootstrap version

**Comment by zufrieden** [comment]
Linked to #241 updating Bootstrap

**Comment by tonifisler** [comment]
Bootstrap has been updated in 4d03fb2. This should be fixed now.

**Comment by tonifisler** [comment]
Need confirmation. All Bootstrap recommendations are already in place.
